examples/golang/docker-compose.yaml

```diff
@@ -0,0 +1,17 @@
+version: "3.8"
+services:
+
+  proxy:
+    build:
+      context: .
+      dockerfile: Dockerfile-proxy
+    depends_on:
+    - web_service
+    ports:
+    - "8080:8080"
+
+  web_service:
+    environment:
+      - PORT=9000
```

Comment:
im confused this didnt fail yaml linting

Reply:
fixed.